### PR TITLE
fix bug 1209564 - Finish New Relic configuration

### PR DESCRIPTION
**Pull request by jwhitlock**
* Run celery under New Relic as well.
* Heroku gets the configuration from the environment, so remove newrelic.ini, which would override it.
* Add a script to record deployment to New Relic
* Add link to beta server docs on github

### CODE REVIEW

**Comment by jwhitlock** [comment]
* Rebased
* Changed error message in ``tools/newrelic_deployment.sh`` if API key is not set
* https://browsercompat.herokuapp.com is no longer running this branch.

**Comment by groovecoder** [comment]
What invokes `newrelic_deployment.sh` ?

**Comment by jwhitlock** [comment]
Heroku doesn't allow setting headers from curl, so can't be done as part of Heroku deployment.  Github doesn't have a post-push commit, so can't be done from git.  So, it's still manual:  https://github.com/mdn/browsercompat/wiki/Beta-Server#deployment

File: tools/newrelic_record_deployment.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+set -e  # Exit on non-zero status from command
+set -u  # Unset variables are error
+set -o pipefail  # Fail on non-zero exit in a pipe
+IFS=$'\n\t'  # Stricter for loops
+
+# Defaults from environment
+API_KEY=${NEW_RELIC_API_KEY:-}
+APP_ID=${NEW_RELIC_DEPLOY_APP_ID:-}
+BRANCH=${NEW_RELIC_BRANCH:-remotes/browsercompat/master}
+DEPLOY_USER=${NEW_RELIC_DEPLOYER:-}
```
Comment (groovecoder) [inline tools/newrelic_record_deployment.sh]: nice. `:-}`

**Comment by groovecoder** [comment]
Are you using the Heroku + GitHub integration for deployments? Last app I used that with automatically sent NR pings on deploys.

r+ in any case

**Comment by jwhitlock** [comment]
It works for one application per account, and I think someone already claimed it for the Mozilla Heroku account: https://docs.newrelic.com/docs/accounts-partnerships/partnerships/heroku-new-relic/heroku-set-deployment-notifications

**Pull request merged**